machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -89,7 +89,6 @@ const model = createModel(
       DOWNLOAD_READY: () => ({}),
       FAILED: () => ({}),
       GET_VC_RESPONSE: (vc: VC) => ({vc}),
-      VERIFY: () => ({}),
```

Comment:
Why is this VERIFY event from here?

Reply:
This event is unused and might have been used for any past features before.

Reply:
Can you remove it from EsignetMosipVCItemMachine as well?

Reply:
Removed the event.